src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,39 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int AGE_MIN_VALUE = 18;
+    public static final int PASSWORD_MIN_LENGTH = 6;
+    private final StorageDaoImpl storageDaoImpl = new StorageDaoImpl();
```

Comment:
You should better use the interface here instead of the implementation

```suggestion
    private final StorageDao storageDao = new StorageDaoImpl();
```

and initialise it in constructor instead of field initialisation to make class more friendly for unit testing.

Reply:
@leonidlist already fixed, thanks, guys <3.